src/WallGo/Config/WallGoDefaults.ini

```diff
@@ -31,6 +31,10 @@ pressRelErrTol = 0.1
 # Maximum number of iterations for the convergence of the pressure
 maxIterations = 20
 
+# Flag to enforce conservation of energy and momentum. 
```

Comment:
Why not True and False? And can this comment be a bit more informative? Something like the following?

Flag to enforce conservation of energy and momentum in the 
fluid and temperature profile. If True, EM-conservation is conserved,
if False, it is not. Normally this should be set to True, but if all particles
are tracked in the Boltzmann equations, it should be set to False.

Reply:
For now it's easier to have 0 or 1 because the config file is string-based. I might change that when I update the configs.